File: test/cs/case16_instance_member.cs
using System.Collections.Generic;
using System.IO;
using System;

public static class Program
{
    public class Point
    {
        public int x;
        public int y;

        public Point(int x, int y)
        {
            this.x = x;
            this.y = y;
        }
        public int total()
        {
            return (this.x + this.y);
        }
    }

    public static void Main(string[] args)
    {
        Point p = new Point(2, 5);
        Pytra.CsModule.py_runtime.print(p.total());
    }
}

File: test/py/case16_instance_member.py
class Point:
    def __init__(self, x: int, y: int) -> None:
        self.x = x
        self.y: int = y

    def total(self) -> int:
        return self.x + self.y


if __name__ == "__main__":
    p: Point = Point(2, 5)
    print(p.total())
